drivers/SmartThings/matter-lock/src/new-matter-lock/init.lua

```diff
@@ -1685,7 +1752,10 @@ local new_matter_lock_handler = {
     capabilities.lock,
     capabilities.lockUsers,
     capabilities.lockCredentials,
-    capabilities.lockSchedules
+    capabilities.lockSchedules,
+    capabilities.remoteControlStatus,
```

Comment:
There is no handler for `remoteControlStatus`, should there be or should this be removed?

Reply:
I don't think it's needed. I will remove it.